docs/userGuide/syntax/mathformulae.mbdf

```diff
@@ -0,0 +1,99 @@
+## Math Formulae
+Markbind supports typesetting TeX math equations. [KaTeX](https://katex.org) is used as a fast math renderer.
+
+<box type="info">
+
+Insert `inline` equations by enclosing them in round brackets `\( ... \)`.
```

Comment:
would it better to bold the **inline/display** or link it somewhere? let's use the inline code syntax just for code